src/nodes.coffee

```diff
@@ -4927,8 +4923,20 @@ exports.StringWithInterpolations = class StringWithInterpolations extends Base
           element.originalValue
           tail: element is last
         ).withLocationDataFrom(element).ast o
-      else
-        expressions.push element?.unwrap().ast(o) ? null
+      else # Interpolation
+        {expression} = element
+        node =
+          unless expression?
+            emptyInterpolation = new EmptyInterpolation()
+            emptyInterpolation.locationData = emptyExpressionLocationData {
+              interpolationNode: element
+              openingBrace: '#{'
+              closingBrace: '{'
```

Comment:
Not `}`?

Reply:
Good catch - it happened not to be breaking because it just uses the length of the brace string

Updated to `'}'`